app/queries/keywords_query.rb

```diff
@@ -9,14 +9,28 @@ def initialize(scope, filter_params)
   end
 
   def call
-    return scope unless defined?(filter_params[:adwords_url_contains]) && filter_params[:adwords_url_contains]
-
-    filter_ads_top_urls(filter_params[:adwords_url_contains])
+    if filter_params[:adwords_url_contains]
```

Comment:
Just minor suggestion, we can return if early to reduce the nested level.

```suggestion    
      return filter_ads_top_urls(filter_params[:adwords_url_contains]) if filter_params[:adwords_url_contains]
      return filter_result_urls(filter_params[:word], filter_params[:match_at_least]) if filter_params[:word] && filter_params[:match_at_least]

      scope
    end
```